src/encoded/tests/test_batch_download.py

```diff
@@ -423,34 +386,42 @@ def test_batch_download_lookup_column_value(lookup_column_value_item, lookup_col
-    ('/metadata/type=Experiment&assembly=hg19&assembly=GRCh38&format=json/metadata.tsv', 'ENCFF000VUS\tbam\talignments\tENCSR000ACY\tDNA methylation profiling by array assay\tCL:1000350\tbasal cell of epithelium of terminal bronchiole\tprimary cell\tHomo sapiens\t\t\t\t\t\t\t\t\t\t\tDNA\t\tQIAGEN DNeasy Blood & Tissue Kit\tQIAGEN DNeasy Blood & Tissue Kit\t\tFalse\t2016-01-01\tENCODE\t\t\t\t2\t1\t\t\t\t\t\t\t473944988\tMichael Snyder, Stanford\t91d0dd9e0df439dd6599914b5275e7b2\t\thttp://localhost/files/ENCFF000VUS/@@download/ENCFF000VUS.bam\thg19\t\t\t\treleased\t\t\tmismatched status, missing derived_from\t\tinconsistent replicate', 'SRR1270627\tsra\treads\tENCSR765JPC\twhole-genome shotgun bisulfite sequencing\tEFO:0002067\tK562\tcell line\tHomo sapiens\t\t\t\t\t\t\t\t\t\t\tDNA\t\t\t\t\tFalse\t2016-01-01\tENCODE\t\t\t\t1\t1\t36\t\tsingle-ended\t\t\t\t1247813024\tRichard Myers, HAIB\tb52abba2d1e08ea93c527e38aff96e11\tSRA:SRR1270627\thttp://localhost/files/SRR1270627/@@download/SRR1270627.sra\t\t\tIllumina Genome Analyzer IIx\t\treleased\t\t\tmismatched status, mismatched file status, experiment not submitted to GEO\tunreplicated experiment, missing documents, insufficient read length\t',),
-    ('/metadata/type=Annotation&organism.scientific_name%21=Homo+sapiens&organism.scientific_name=Mus+musculus&format=json/metadata.tsv','ENCFF015OKV\tbed enhancer predictions\tpredicted heart enhancers\tENCSR356VQT\tlong-range chromatin interactions\tSamtools, Picard\t2\tUBERON:0000948\theart\ttissue\tembryonic\t11.5\tday\tMus musculus\t\t2015-02-27\tENCODE\tJ. Michael Cherry, Stanford\ta5f849c78025c80dca58771492c97318\t\thttp://localhost/files/ENCFF015OKV/@@download/ENCFF015OKV.bed.gz\tmm10\t\treleased\t\t\t1544154147\tmissing analysis_step_run\tmismatched status, missing derived_from','File accession\tFile format\tOutput type\tDataset accession\tAnnotation type\tSoftware used\tEncyclopedia Version\tBiosample term id\tBiosample term name\tBiosample type\tLife stage\tAge\tAge units\tOrganism\tTargets\tDataset date released\tProject\tLab\tmd5sum\tdbxrefs\tFile download URL\tAssembly\tControlled by\tFile Status\tDerived from\tS3 URL\tSize\tAudit WARNING\tAudit INTERNAL_ACTION\tAudit NOT_COMPLIANT\tAudit ERROR\r',),
```

Comment:
and these?

Reply:
This also aren't quoted correctly: `/metadata/type=Experiment&biosample_ontology.term_name!=basal cell of epithelium of terminal bronchiole&format=json/metadata.tsv` All the links pass in URL encoded URLs. These have blank spaces in them.